app/controllers/browse_controller.rb

```diff
@@ -2,7 +2,11 @@
 
 class BrowseController < ApplicationController
   def index
-    @shelf_list = ShelfListPresenter.new(shelf_list_params)
+    @shelf_list = ShelfListPresenter.new(lc_shelf_list_params)
+  end
+
+  def dewey
+    @shelf_list = ShelfListPresenter.new(dewey_shelf_list_params)
```

Comment:
Take a look at https://github.com/psu-libraries/psulib_blacklight/pull/891. I think it'll be easier if we treat classification as a parameter, ex: `/browse/call_numbers?classification=lc`. However, we might want to make it a kind of enum.

Reply:
I have added changes to make it a param.